### PR TITLE
apifctl fixes

### PR DESCRIPTION
**Pull request by kimsauce**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/api-testing/apifctl-cicd-integration.md [hunk 1]
```diff
@@ -203,6 +211,15 @@ Available Options:
 * [<code>-l &#60;limit&#62;</code>](#-l-limit) <small>| OPTIONAL | INTEGER |</small>
 * [<code>-o &#60;offset&#62;</code>](#-o-offset) <small>| OPTIONAL | INTEGER |</small>
 
+```bash title=Full Example"
+docker run quay.io/saucelabs/apifctl event \
```
Comment (LoryApiFortress) [inline docs/api-testing/apifctl-cicd-integration.md:215]: @kimsauce typo: replace "event" with "events" (docker run quay.io/saucelabs/apifctl events"

File: docs/api-testing/apifctl-cicd-integration.md [hunk 2]
```diff
@@ -365,18 +390,17 @@ Identifies the end date of the events you want to see. For use with **[`metrics`
 ### `-tag <tag(s)>`
 <p><small>| STRING |</small></p>
 
-The **[`run-tag`](#run-tag)** command will run all the tests in your Project marked with that tag. Format as a comma-separated list of tags you want to assign to the test. For use with **[`run-tag`](#run-tag)** command only.
+The **[`run-tag`](#run-tag)** command will run all the tests in your Project marked with that tag. For use with the run-tag command only. For use with the **[`run-tag`](#run-tag)** command only. The example below will run all the tests in the Project labeled with the `product` and `production` tags.
```
Comment (LoryApiFortress) [inline docs/api-testing/apifctl-cicd-integration.md:393]: @kimsauce you said two times the same thing ("For use with the run-tag command only. For use with the **[`run-tag`](#run-tag)** command only." ) I guess one is enough :)

Comment (kimsauce) [reply docs/api-testing/apifctl-cicd-integration.md:393]: yikes!! lol good catch.

File: docs/api-testing/apifctl-cicd-integration.md [hunk 3]
```diff
@@ -152,7 +152,10 @@ Available Options:
 ```bash title="Full Example"
 docker run -v $(pwd)/tests:/tests quay.io/saucelabs/apifctl upload \
 -H https://<username>:<accessKey>@api.us-west-1.saucelabs.com/api-testing/rest/v4/<hookId> \
--p /tests/test_abc
```
Comment (LoryApiFortress) [inline docs/api-testing/apifctl-cicd-integration.md:155]: @kimsauce why did you add "-v $(pwd)/tests:/tests " (lines 129 and 153)? keep it simple as the other examples
